transit_odp/templates/api/avl_subscriptions_subscribe.html

```diff
@@ -0,0 +1,31 @@
+{% extends "page.html" %}
+{% load static %}
+{% load breadcrumbs %}
+{% load i18n %}
+{% load crispy_forms_tags %}
+{% block title_tag %}
+  {% if form.errors %}
+    {% trans "Error: " %}
+  {% endif %}
+  {% trans "Create AVL Data Subscription" %}
```

Comment:
```suggestion
  {% trans "Create Location Data Subscription" %}
```

We discussed in standup about calling this API "Location data" rather than "AVL data" as it's more user friendly. I would update the code and leave a note for Anna to update the designs. Think breadcrumb naming will also need updating too :)